### PR TITLE
Rename InteractiveEditingSession to GraphicalEditingScope

### PR DESCRIPTION
**Pull request by pmconne**

### CODE REVIEW

File: core/frontend/src/GraphicalEditingScope.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  /** The connection to the iModel being edited. */
+  public readonly iModel: BriefcaseConnection;
+
+  /** Event raised when a new scope is created for any [[BriefcaseConnection]].
+   * @see [[onExiting]] and [[onExited]] for complementary events.
+   */
+  public static readonly onEnter = new BeEvent<(scope: GraphicalEditingScope) => void>();
+
+  /** Event raised when this scope is about to exit.
+   * @see [[onBEnteregin]] for the complementary event.
```
Comment (kabentley) [inline core/frontend/src/GraphicalEditingScope.ts:72]: typo

File: core/common/src/IpcAppProps.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ export enum IpcAppChannel {
   Functions = "ipc-app",
   AppNotify = "ipcApp-notify",
   Txns = "txns",
-  EditingSession = "editing-session",
+  EditingScope = "editing-session",
```
Comment (kabentley) [inline core/common/src/IpcAppProps.ts:57]: "editing-scope"

**Review by kabentley — APPROVED** [review]

File: core/frontend/src/GraphicalEditingScope.ts (hunk 1 above)
Comment (pmconne) [reply core/frontend/src/GraphicalEditingScope.ts:72]: @williamkbentley do you understand why bmsBuild did not catch this?

**Comment by pmconne** [comment]
@calebmshafer all checks had passed and changes were approved. I pushed 1 minor commit to fix typos. Status did not change, still showed all checks passed, did not trigger new builds. I closed and reopened, and 2 of the checks immediately showed as "failed". Their details links go to builds with no output. I'm going to copy to a new branch.

**Comment by calebmshafer** [comment]
> @calebmshafer all checks had passed and changes were approved. I pushed 1 minor commit to fix typos. Status did not change, still showed all checks passed, did not trigger new builds. I closed and reopened, and 2 of the checks immediately showed as "failed". Their details links go to builds with no output. I'm going to copy to a new branch.

@pmconne sounds like ADOps lost the association or the web hook.  Let me know if this happens again, I can dig further.

**Pull request closed without merging**